pages/loginPage.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://splash-ui.vercel.app/CSS/main.css">
+    <link rel="stylesheet" href="/css/loginPage.css">
+    <title>Sneaker Street</title>
+</head>
+<body>
+    <header class="header-main">
+        <nav class="nav nav-dark" style="position: relative;">
```

Comment:
instead of inline styles, define your own custom CSS utility class and use it here

Reply:
Okay will check that